src/main/java/com/redhat/cloud/policies/engine/process/EventProcessor.java

```diff
@@ -59,7 +63,10 @@ public void process(Event event) {
         } else {
             LOGGER.debugf("Found %d enabled policies for account %s", enabledPolicies.size(), event.getAccountId());
 
-            PoliciesAction policiesAction = new PoliciesAction();
+            PoliciesAction policiesAction = new PoliciesAction(orgIdConfig.isUseOrgId());
+
+            policiesAction.setOrgId(event.getAccountId());
```

Comment:
This should be:
```suggestion
            policiesAction.setOrgId(event.getOrgId());
```
but first Event needs to be updated with the `orgId` value retrieved in `PayloadParser`.